src/mscorlib/model.xml

```diff
@@ -1825,6 +1825,22 @@
       <Member Name="FailFast(System.String,System.Exception)"  />
       <Member Name="Exit(System.Int32)" />
     </Type>
+    <Type Name="Internal.Runtime.Augments.EnvironmentAugments">
```

Comment:
> we support using, for example, a 1.0 corefx (and in this specific case a 1.0 System.Runtime.Extensions.dll) with a 1.1 coreclr

1.0 corefx assemblies that depend on public contracts only should be supported with a 1.1 CoreCLR. For example, `System.Reflection.Metadata`.